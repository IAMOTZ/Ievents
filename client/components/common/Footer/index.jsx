import React from 'react';

const footerStyle = {
  backgroundColor: 'rgba(0,0,0, 0.9)',
  borderTop: '2px solid rgb(0, 142, 214)'
};

const Footer = () => (
  <footer className="w-100 mt-auto" style={footerStyle}>
    <div className="container text-white text-center py-5">
      <h1>iEvents</h1>
      <p>Copyright &copy; 2017</p>
    </div>
  </footer>
);

export default Footer;
