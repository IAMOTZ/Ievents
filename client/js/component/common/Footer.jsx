import React from 'react';

const Footer = () => (
  <footer className="w-100 mt-auto">
    <div className="container text-white text-center py-5">
      <h1>Ievents</h1>
      <p>Copyright &copy; 2017</p>
    </div>
  </footer>
);

export default Footer;
