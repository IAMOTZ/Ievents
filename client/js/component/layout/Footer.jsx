import React from 'react';

export default () => {
  return (
    <footer className="w-100 mt-auto">
      <div className="container text-white text-center py-5">
        <h1>Ievents</h1>
        <p>Copyright &copy; 2017</p>
      </div>
    </footer>
  );
};
