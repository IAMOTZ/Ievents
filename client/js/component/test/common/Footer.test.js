import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../common/Footer.jsx';

describe('<Footer />', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
