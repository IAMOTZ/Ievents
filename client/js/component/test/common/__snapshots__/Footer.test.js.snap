// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<footer
  className="w-100 mt-auto"
>
  <div
    className="container text-white text-center py-5"
  >
    <h1>
      Ievents
    </h1>
    <p>
      Copyright Â© 2017
    </p>
  </div>
</footer>
`;
